packages/liferay-npm-scripts/src/utils/generate-soy-dependencies.js

```diff
@@ -23,6 +26,7 @@ module.exports = function(dependencies) {
 			return resolvedDependency;
 		})
 		.filter(Boolean)
+		.filter(dependencyPath => dependencyPath !== cwd)
```

Comment:
I'm adding another pass, just to make it explicit, but we should probably look at ways to optimize this.